packages/docs-page/index.js

```diff
@@ -17,6 +17,7 @@ import generateComponents from './components'
 import temporary_injectJumpToSection from './temporary_jump-to-section'
 import LoadingSkeleton from './components/loading-skeleton'
 import useIsMobile from './use-is-mobile'
+import s from './style.module.css'
```

Comment:
Have converted `docs-page` to CSS modules, as I would need to run a release anyways due to the underlying `content` changes.